src/core/core.datasetController.js

```diff
@@ -268,6 +268,39 @@ helpers.extend(DatasetController.prototype, {
 		}
 	},
 
+	/**
+	 * Returns a set of predefined style properties that should be used to represent the dataset
+	 * or the data if the index is specified
+	 * @params {number} index - data index
+	 * @return {IStyleInterface} style object
+	 */
+	getStyle: function(index) {
+		var me = this;
+		var meta = me.getMeta();
+		var element, style;
+
+		if (meta.dataset && index === undefined) {
+			element = meta.dataset;
+		} else {
+			index = index || 0;
+			element = meta.data[index];
+		}
+		style = me._resolveElementOptions(element || {}, index);
+
+		if (style.fill === false || style.fill === null) {
+			style.backgroundColor = 'rgba(0,0,0,0)';
+		}
+
+		return style;
+	},
+
+	/**
+	 * @private
+	 */
+	_resolveElementOptions: function() {
+		throw new Error('This method is not implemented.');
```

Comment:
Isn't it a breaking change for custom (external) controllers inheriting the dataset controller directly, thus would not have this method implemented? Should we provide a base implementation that would resolve only common options?

Reply:
Implemented this in the datasetController.